### ISSUE TITLE
No longer displays on desktop - Chrome 89+

### ISSUE THREAD

**Issue by aamir-madari** [on2-dev/share-api-polyfill#69]
The support for the Web Share API has been added to desktop as of Chrome 89. Therefore the polyfill no longer displays on desktop in Chrome/Edge. 

The issue is the Windows equivalent isn't as great as it is on iOS and Android and lacks the ability to share on social media like in the polyfill. 

Is there any way of utilising the polyfill rather than the native API? Can it be manually evoked via JS?

**Comment by valentinoli** [comment]
I don't think there is an option for that. As [you can see here](https://github.com/on2-dev/share-api-polyfill/blob/43c8bf027a51660a661cca03b8fca9b1cde32e87/src/share.js#L6) the polyfill is a fallback to `navigator.share`. So if `navigator.share` exists, then the browser's implementation is used and not the polyfill.

**Comment by valentinoli** [comment]
I agree that the Web Share feature in recent Edge and Chrome versions is weird, there doesn't seem to be an option for sharing on social media which is arguably the most popular way to use it nowadays.

**Comment by valentinoli** [comment]
You might be able to do

```js 
navigator.share = undefined
```

before importing the polyfill.
